src/clamd/__init__.py

```diff
@@ -63,8 +63,8 @@ def _init_socket(self):
         """
         try:
             self.clamd_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            self.clamd_socket.connect((self.host, self.port))
```

Comment:
Docs please :)

Reply:
Can you explain what you think should be documented? The fact that the existing code calls `connect` before `settimeout` seems more like an oversight than anything else.

Reply:
Why does the order matter?

Reply:
Well I mean... it is explicitly stated in the python docs: https://docs.python.org/3/library/socket.html#timeouts-and-the-connect-method

You think we should comment in this repo as well?

Reply:
Your call :) Usually when something trips me up, I write a comment about it.